src/main/java/io/github/fusionflux/portalcubed/content/PortalCubedBlocks.java

```diff
@@ -115,6 +136,135 @@ public class PortalCubedBlocks {
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_GRAY)
+			)
+			.build();
+	public static final RotatedPillarBlock STRIPPED_LEMON_WOOD = REGISTRAR.blocks.create("stripped_lemon_wood", RotatedPillarBlock::new)
+			.copyFrom(Blocks.OAK_WOOD)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final LeavesBlock LEMON_LEAVES = REGISTRAR.blocks.create("lemon_leaves", LeavesBlock::new)
+			.copyFrom(Blocks.OAK_LEAVES)
+			.build();
+	public static final SaplingBlock LEMON_SAPLING = REGISTRAR.blocks.create("lemon_sapling", settings -> new SaplingBlock(PortalCubedFeatures.LEMON_TREE_GROWER, settings))
+			.copyFrom(Blocks.OAK_SAPLING)
+			.renderType(RenderTypes.CUTOUT)
+			.build();
+	public static final FlowerPotBlock POTTED_LEMON_SAPLING = REGISTRAR.blocks.create("potted_lemon_sapling", settings -> new FlowerPotBlock(LEMON_SAPLING, settings))
+			.copyFrom(Blocks.POTTED_OAK_SAPLING)
+			.item(BlockItemProvider::noItem)
+			.renderType(RenderTypes.CUTOUT)
+			.build();
+	public static final Block LEMON_PLANKS = REGISTRAR.blocks.create("lemon_planks", Block::new)
+			.copyFrom(Blocks.OAK_PLANKS)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final SlabBlock LEMON_SLAB = REGISTRAR.blocks.create("lemon_slab", SlabBlock::new)
+			.copyFrom(Blocks.OAK_SLAB)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final SaneStairBlock LEMON_STAIRS = REGISTRAR.blocks.create("lemon_stairs", SaneStairBlock::new)
+			.copyFrom(Blocks.OAK_STAIRS)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final FenceBlock LEMON_FENCE = REGISTRAR.blocks.create("lemon_fence", FenceBlock::new)
+			.copyFrom(Blocks.OAK_FENCE)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final FenceGateBlock LEMON_FENCE_GATE = REGISTRAR.blocks.create("lemon_fence_gate", properties -> new FenceGateBlock(WoodType.OAK, properties))
+			.copyFrom(Blocks.OAK_FENCE_GATE)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final ButtonBlock LEMON_BUTTON = REGISTRAR.blocks.create("lemon_button", properties -> new ButtonBlock(BlockSetType.OAK, 30, properties))
+			.copyFrom(Blocks.OAK_BUTTON)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	public static final PressurePlateBlock LEMON_PRESSURE_PLATE = REGISTRAR.blocks.create("lemon_pressure_plate", properties -> new PressurePlateBlock(BlockSetType.OAK, properties))
+			.copyFrom(Blocks.OAK_PRESSURE_PLATE)
+			.settings(settings -> settings
+					.mapColor(MapColor.TERRACOTTA_YELLOW)
+			)
+			.build();
+	private static final ResourceLocation LEMON_SIGN_TEXTURE = PortalCubed.id("entity/signs/lemon");
```

Comment:
These shouldn't be stored here. also the name doesn't match the access

Reply:
Where should they be stored

Reply:
I guess it's fine. make them public though